content/blog/gsoc-25-whois-and-ip-geolocation.rst

```diff
@@ -0,0 +1,226 @@
+WHOIS Information and IP Address-Based Geolocation
```

Comment:
What do you think of?

```suggestion
GSoC 2025: WHOIS Info and Estimated Geographic Locations
```

I'd like to make sure all blog posts are prefixed with "GSoC 2025".